pino.js

```diff
@@ -338,6 +343,15 @@ function pino (opts, stream) {
     }
   }
 
+  const baseLog = typeof iopts.baseLog === 'object' ? iopts.baseLog : defaultBaseLog
```

Comment:
The usage of `const` is inconsistent with the rest of the codebase. Please change this to `var`. Also, I'd like to see some grouping here to make the ternary easier to read.